bundlesize.config.json

```
@@ -2,7 +2,7 @@
   "files": [
     {
       "path": "packages/autocomplete-core/dist/umd/index.production.js",
-      "maxSize": "6 kB"
+      "maxSize": "6.2 kB"
```

We grow this property by 0.25 increments.

```suggestion
      "maxSize": "6.25 kB"
```